07-map.slide

```diff
@@ -2,19 +2,148 @@ Maps in Go
 22 Feb 2017
 Tags: edmontongo, golang, workshop
 
+Lukasz Rozycki
+[[https://lukaszr.com]]
+
 * Maps
 
-composite literal for creating maps
+- Hash Table, Dictionary
+- Mapping from key to a value
+- Fast lookups, adds, and deletes
+- A Go map type looks like this:
+ map[KeyType]ValueType
+- Maps are not safe for concurrent use
+
+* What can be a key?
+
+- Any type that is comparable
+ ==    equal
+ !=    not equal
+- boolean, numeric, string, pointer
+- interface type, channel
```

Comment:
since we aren't covering concurrency, may not want to mention channels here... it's sure to lead to questions and take people off track